web-registry/src/pages/Marketplace/Storefront/Storefront.tsx

```diff
@@ -28,6 +33,11 @@ export const Storefront = (): JSX.Element => {
     [sellOrders],
   );
   const batchInfos = useQueryListBatchInfo(batchDenoms);
+  // offchain stored Projects
+  const { data: offChainProjects } = useAllProjectsQuery();
```

Comment:
I assumed here that all projects on graphql were off-chain projects but I'm not sure about this.
Can change the naming if needed.

Reply:
Seems like a fine distinction (could call it `offChainProjectData` if you prefer) - there might be some crossover on fields from legacy projects that don't have on-chain data (IE a project could be both on-chain and in the DB) in which case we should probably normalize to the ledger data.